addons/medical_treatment/XEH_postInit.sqf

```diff
@@ -21,3 +21,60 @@ if (isServer) then {
 [QGVAR(actionCheckPulseLocal), FUNC(actionCheckPulseLocal)] call CBA_fnc_addEventHandler;
 [QGVAR(actionCheckBloodPressureLocal), FUNC(actionCheckBloodPressureLocal)] call CBA_fnc_addEventHandler;
 [QGVAR(actionPlaceInBodyBag), FUNC(actionPlaceInBodyBag)] call CBA_fnc_addEventHandler;
+
+// Notification lets the player known if they're being treated
+[QGVAR(initiated), {
+    params ["_caller", "_target"];
+
+    // Notifications don't make sense for self-healing or when dead
+    if (_target != ACE_player || {_target == _caller} || {!alive _target}) exitWith {};
+
+    // Can't tell who is providing aid when you're unconscious
+    private _name = if IS_UNCONSCIOUS(_target) then {
+        LLSTRING(UnknownPerson);
+    } else {
+        name _caller
+    };
+
+    QGVAR(notification) cutText [format [LLSTRING(Notification), _name], "PLAIN DOWN"];
+
+    // We track who is currently treating the unit
+    private _responders = _target getVariable [QGVAR(responders), []];
+    _responders pushBack _caller;
+    _target setVariable [QGVAR(responders), _responders];
+}] call CBA_fnc_addEventHandler;
+
+// Notification may no longer apply when treatment ends
+private _treatmentEnded = {
+    params ["_caller", "_target"];
+
+    // Notifications don't make sense for self-healing or when dead
+    if (_target != ACE_player || {_target == _caller} || {!alive _target}) exitWith {};
```

Comment:
in that case, it should wrap the notifications logic in this conditional. I see a bunch of other logic that may be used for more generic purposes that should run regardless of this condition. i.g. the `deleteAt` part.

Reply:
`deleteAt` doesn't need to run for dead or self healing since self healing doesn't add the unit and in the first place and upon death the whole array is deleted. However, I'll split out the player check and put that before the notification